data/categories.txt

```diff
@@ -2298,6 +2303,7 @@ el:Αγορά|αγορά|κατάστημα
 sk:3Trhovisko|samoobsluha
 sw:Soko
 fa:بازار
+ca:Mercat
```

Comment:
Thanks! Can you pleas try to sort added translations in all places?

Reply:
Thank you for the advice, I ordered all ca strings